docs/sphinx/developers/cpp/overview.txt

```diff
@@ -768,6 +769,18 @@ may also be specified.  Please see the :program:`cmake` documentation
 for further details of all configurable options, and run ``cmake
 --help`` to list the available generators for your platform.
 
+If using the superbuild:
+
+source-cache=directory
+  Specify a directory in which to store downloaded source files; this
+  is useful if you need to repeat the build since the source files
+  will not need downloading again.
+bioformats-superbuild_USE_SYSTEM_${package}=(ON|OFF)
+  Disable the building of particular components, in order to use the
+  system version of these components.  By default, all components are
+  enabled. `${package}` is the component name.  Look in the
```

Comment:
Maybe s/enabled/built/? Just wondering how clear it is that we're talking about enabling the building of the components rather than enabling the component at all.